metrics/core/metrics.go

```diff
@@ -179,6 +179,16 @@ var MetricUptime = Metric{
 	},
 }
 
+var MetricRestartCount = Metric{
```

Comment:
This doesn't belong to this PR, this is not about supporting the new resource model

Reply:
For new resources we have different set of metric. I don't think this PR can be sensibly split into "using new resource model" and "using new set of metrics".

Reply:
This is not an old metric in a new format, this is a totally new metric that never existed and can be added in a separate PR

Reply:
Done

Reply:
Done.